docs/guidelines.md

```diff
@@ -0,0 +1,87 @@
+# Feedback guidelines
+
+The goal of a language track on Exercism is to give people a way to achieve a
+high level of fluency at a low level of proficiency. We're aiming for fluency
+in the syntax, idioms, and the standard library of the language. You can read
+more about the [goal of exercism here](https://github.com/exercism/docs/blob/master/about/goal-of-exercism.md).
+
+## Definitions
+
+In the following paragraphs, keywords such as **MUST**, **SHOULD**, **COULD**
+are to be interpreted as in [RFC2119](https://www.ietf.org/rfc/rfc2119.txt);
+given that we recognise the following four output states and their restrictions:
+
+- `approve_as_optimal`: **COULD** be a best solution, **SHOULD** be without comment
+- `approve_with_comment`: **COULD** be a best solution, **MUST** be with comment
+- `disapprove_with_comment`: **MUST** be with comment
+- `refer_to_mentor`: **COULD** be with comment
+
+## Conditions and outcome
+
+For feedback generated by automated mentoring:
+
+- it **SHOULD** `disapprove_with_comment` when someone isn't using one of the
+  subjects the exercise is supposed to teach,
+- each `disapprove_with_comment` **MUST** have at least one comment,
+- each `disapprove_with_comment` **SHOULD** steer a student towards an `approve_`,
+- a student **SHOULD NOT** get a `refer_to_mentor` in a newer iteration, if they've
+  had a `disapprove_with_comment` in a previous iteration,
+- a `refer_to_mentor` **COULD** have one or more comments which will be given to
```

Comment:
So these comments will only be passed to the mentor, not the student?

Reply:
TBD -- currently no comments will be passed to anyone 🚫. This will be up to the product team ;)

I wrote it like this because an earlier comment by @iHiD indicated that `refer_to_mentor` would refer the comments to the mentor! Such as the "more than 3 components interpolation" string.

Personally I think it would be _really_ helpful for mentors to understand _why_ it was referred to them.

Reply:
Agreed, although in many cases it will be: "because I (the analyzer) don't understand the code" :)

Reply:
Definitely! But the great thing is that we _could_ collect _what_ it doesn't understand and then iterate. Although that might be better in `analysis.out`

Reply:
I'm happy with this. I have lots of cases where I actively refer to mentor knowing what is weird but not how to fix it. I've acutally decided in lots of places not to tell the student what is wrong even though I could work it out, because I think it'll be better with a human conversation.